.devcontainer/scripts/postCreateCommand.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -e
+
+# reset the repo to the commit hash that was used to build the prebuilt Codespace
+git reset --hard $(cat ./artifacts/prebuild.sha)
```

Comment:
Curious why this is needed.

Reply:
I'm honestly not sure. I stole it from runtime (not saying that's a good justification): https://github.com/dotnet/runtime/blob/main/.devcontainer/scripts/postCreateCommand.sh